schema/encoding/jsonschema/example_test.go

```diff
@@ -35,25 +34,25 @@ func ExampleEncoder() {
 	json.Indent(b2, b.Bytes(), "", "| ")
 	fmt.Println(b2)
 	// Output: {
-	// | "type": "object",
 	// | "additionalProperties": false,
 	// | "properties": {
 	// | | "bar": {
 	// | | | "type": "integer"
 	// | | },
 	// | | "baz": {
+	// | | | "maxLength": 42,
 	// | | | "readOnly": true,
-	// | | | "type": "string",
-	// | | | "maxLength": 42
+	// | | | "type": "string"
 	// | | },
 	// | | "foo": {
-	// | | | "type": "number",
-	// | | | "minimum": 0E+00
+	// | | | "minimum": 0,
+	// | | | "type": "number"
 	// | | },
 	// | | "foobar": {}
```

Comment:
Updated the example to include a new description @yanfali. Hopefully it's at least a bit more clear from the example what's going on now...